litmus-portal/frontend/src/components/WelcomeModal/styles.ts

```diff
@@ -4,22 +4,28 @@ const useStyles = makeStyles((theme: Theme) => ({
   root: {
     '& > *': {
       margin: theme.spacing(1),
-      // width: 350,
       outline: 'none',
       marginTop: 30,
     },
   },
   rootContainer: {
-    marginLeft: '21rem',
-    marginRight: '21rem',
-    marginTop: '7.8rem',
-    marginBottom: '7.8rem',
-    paddingBottom: '6rem',
+    marginLeft:  theme.spacing(37),
+    marginRight:  theme.spacing(37),
+    marginTop: '5rem',
```

Comment:
Keeping the height and width properties the other marginTop, Bottom, Left, Right, Padding, should be taken from theme